integration/tests/cook/test_basic.py

```diff
@@ -2634,3 +2634,21 @@ def test_kubernetes_disallowed_volumes(self):
             util.wait_for_instance(self.cook_url, job_uuid, status='success')
         finally:
             util.kill_jobs(self.cook_url, [job_uuid])
+
+    def test_priority(self):
+        _, resp = util.submit_job(self.cook_url,priority=0)
+        self.assertEqual(201, resp.status_code, msg=resp.content)
+
+        _, resp = util.submit_job(self.cook_url,priority=100)
+        self.assertEqual(201, resp.status_code, msg=resp.content)
+
+        _, resp = util.submit_job(self.cook_url,priority=16000000)
+        self.assertEqual(201, resp.status_code, msg=resp.content)
+
+        _, resp = util.submit_job(self.cook_url,priority=-1)
+        self.assertEqual(400, resp.status_code, msg=resp.content)
+        self.assertTrue(f'priority":"(not (between-0-and-16000000' in str(resp.content))
+
+        _, resp = util.submit_job(self.cook_url,priority=16000001)
+        self.assertEqual(400, resp.status_code, msg=resp.content)
+        self.assertTrue(f'priority":"(not (between-0-and-16000000' in str(resp.content))
```

Comment:
Can you run this through the python code formatter?

Reply:
done